django/contrib/gis/db/backends/base/operations.py

```diff
@@ -135,3 +138,24 @@ def get_geometry_converter(self, expression):
             'Subclasses of BaseSpatialOperations must provide a '
             'get_geometry_converter() method.'
         )
+
+    def get_area_att_for_field(self, field):
+        if field.geodetic(self.connection):
+            if self.connection.features.supports_area_geodetic:
+                return 'sq_m'
+            raise NotImplementedError('Area on geodetic coordinate systems not supported.')
```

Comment:
Is this not implemented or not supported (in which case django.db.utils.NotSupportedError may be a better choice)?

Reply:
Probably `NotSupportedError` fits better.

Reply:
Okay, it can be done separately consider this is an existing error message.